cmake/modules/snippets.cmake

```diff
@@ -59,6 +59,7 @@ function(zephyr_process_snippets)
   # Set SNIPPET_ROOT.
   list(APPEND SNIPPET_ROOT ${APPLICATION_SOURCE_DIR})
   list(APPEND SNIPPET_ROOT ${ZEPHYR_BASE})
+  list(APPEND SNIPPET_ROOT ${NRF_DIR})
```

Comment:
This should be using the module variable, not a local variable that might not even exist in this scope. `ZEPHYR_NRF_MODULE_DIR`

Reply:
Thank you, updated.